lib/instrumentation/modules/apollo-server-core.js

```diff
@@ -7,7 +7,7 @@ const clone = require('shallow-clone-shim')
 module.exports = function (apolloServerCore, agent, { version, enabled }) {
   if (!enabled) return apolloServerCore
 
-  if (!semver.satisfies(version, '^2.0.2')) {
+  if (!semver.satisfies(version, '^2.0.2') && !semver.satisfies(version, '^3.0.0')) {
```

Comment:
nit: could be done in a oner

```suggestion
  if (!semver.satisfies(version, '^2.0.2 || ^3.0.0')) {
```